sdk/Cargo.toml

```diff
@@ -165,10 +165,16 @@ web-sys = { version = "0.3.58", features = [
 anyhow = "1.0.40"
 mockall = "0.11.2"
 c2pa = { path = ".", features = [
+	# allow integration tests to use the new API
 	"unstable_api",
-] } # allow integration tests to use the new API
+] }
 jumbf = "0.4.0"
-
+insta = { version = "1.39.0", features = ["json", "redactions", "filters"] }
+# The deps below are used for tests/compat.rs
+tiny_http = "0.12.0"
+bsdiff = "0.2.0"
+# TODO: do we want to use a compression algo already used in the spec?
+lz4_flex = "0.11.3"
```

Comment:
probably a good idea to use something we use elsewhere, but not critical.

Reply:
We may want to consider using the brotli algorithm as it's used by compressed manifests in the spec.